### DISCUSSION TITLE
Sending to Outlook emails does not work!

### DISCUSSION THREAD

**Discussion by MiguelCPro** [postalserver/postal#3104] [Development discussions]
My server is new but it does send emails to Yahoo, Gmail, but I'm having problems sending emails with Outlook. I need to know how to fix this. Please help.

<img width="1529" alt="p1" src="https://github.com/user-attachments/assets/7ca79589-b2b6-4ce6-a033-2e52b781168c">

**Comment by willpower232** [comment]
If you can resolve the domain name and you're unable to connect then I would presume Microsoft is blocking your server providers IP addresses, perhaps because of other previous users from your server provider.

**Reply by MiguelCPro** [reply]
I bought 5 additional IPs to rotate to solve the IP blocking problem, so I don't understand what could be happening. Are there any solutions?
<img width="463" alt="1" src="https://github.com/user-attachments/assets/853e0d66-df2f-4808-97ba-a9c72e028c2e">
<img width="1788" alt="2 1" src="https://github.com/user-attachments/assets/0eedaa81-e48d-4cbb-9d00-9f0de564d5e9">

**Reply by willpower232** [reply]
I imagine Microsoft would be blocking specific ranges rather than individual IP addresses so adding more from the same server provider would not change the situation. I am afraid the only thing you could do would be to ask Microsoft for clarification, it may be that other users of the same server provider have been annoying for them.

**Reply by MiguelCPro** [reply]
Tried using Office 365 Anti-Spam IP Delist Portal: https://sender.office.com/ and still the problem. Is it possible that this is happening because my VPS server and IP are new, if I warm up the server IP with the sending mail, do you honestly think it could help at all ?

**Reply by willpower232** [reply]
It might but if Microsoft is already blocking the IP addresses and you aren't on any lists (see mxtoolbox.com) then they would probably not be aware of your warming up.

**Reply by MiguelCPro** [reply]
If I create another mail server within Postal, would the issue with Outlook be resolved? Or perhaps by creating a new organization from scratch?. It should be noted that I have Postal installed on us.ovhcloud.com

**Reply by willpower232** [reply]
I doubt it, it would still use the same IP addresses to connect from. You either need to try even more IP addresses or ask Microsoft directly.

**Reply by MiguelCPro** [reply]
Ok my friend, thank you very much.

**Reply by MiguelCPro** [reply]
Excuse me, what is the email or link to communicate directly with Microsoft to explain this problem and help me solve it?

**Reply by sinnbeck** [reply]
They have a form you can fill out. I have used it before and gotten help :)

https://olcsupport.office.com/

**Reply by MiguelCPro** [reply]
Thanks very much friend, you also had the same problem with Microsoft?

**Reply by sinnbeck** [reply]
Happy to help.

Close but not quite. All of mails went spam. Based on the mail header I suspected something was wrong (don't recall exactly what). They told me to follow best practices (a pdf). I told them I did and exactly what kind of mails I was sending (and only our company uses the mail server).

 They came back a few days later telling me they had blacklisted my ip due to it being unknown and that they had removed it from the blacklist based on what I told them. Mails ended up in the inbox after that

**Reply by MiguelCPro** [reply]
When messages are held and then released, do they reach the inbox or do they bounce? and What is the best server and mail heater that you recommend?

<img width="1540" alt="image" src="https://github.com/user-attachments/assets/3cde6042-a308-4e4f-a5fb-3fded019a166">
